tests/acceptance/ColumnsTasksCept.php

```diff
@@ -42,5 +42,5 @@
 $I->see( 'Columns Block description', 'div' );
 $I->see( 'Column 1', '.step-info h5' );
 $I->see( 'Column 1 description', '.step-info p' );
-$I->seeElement( '.steps-action img' );
+$I->seeElement( '.steps-action img, .steps-info img' );
 $I->see( 'Explore', '.steps-action a.btn-secondary' );
```

Comment:
This check is also need to change, if above point is valid.

Reply:
Just made the changes, thank you 👍